### DISCUSSION TITLE
can we have a filter option on products page

### DISCUSSION THREAD

**Discussion by wanghaisheng** [mearashadowfax/ScrewFast#170] [Ideas]
options will come from category or specification like this:

Length (mm)	Various
Weight (g)	N/A
Material	Stainless Steel
Finish	Assorted
Package Contents	Various screws in a set

something like this will be better
![image](https://github.com/user-attachments/assets/3cc25165-90a6-4c15-b1fc-4f00a145da77)

**Comment by mearashadowfax** [comment]
Hi @wanghaisheng! Thank you for your suggestion for enhancing the theme! I'll consider it and look into updating the template accordingly in the future.

**Reply by wanghaisheng** [reply]
this was my best
follow this   https://unsplash.com/s/photos/man
with ai code ,   managed to publish a page on cloudflare
https://dd35c834.screwfast-412.pages.dev/products/
